src/Conversions/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+add_bipedal_locomotion_library(
+    NAME                   CommonConversions
+    IS_INTERFACE
+    PUBLIC_HEADERS         include/BipedalLocomotion/Conversions/CommonConversions.h
+    PUBLIC_LINK_LIBRARIES  MANIF::manif iDynTree::idyntree-core)
```

Comment:
I don't think ``manif`` is needed here

Reply:
Thanks! Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/138/commits/6b86f6471647f4dba12cd34dde1b820bf804b71b